docs/arch/README.md

```diff
@@ -0,0 +1,105 @@
+    "symbols": ["ADS.F", "ALV.F", "BAYN.F",  "BEI.F"],
+    "risk_weight": 10,
+    "esg_weight": 10,
+    "start": "2022-08-01",
+    "end": "2022-10-10"
+}'
+```
+
+This starts the calculations depending on the `models` parameter. Once the service is done calculating the various portfolio models, the results are returned and send to the frontend. The object that is returned may look like the following example:
+
+```JSON
+[
+    {
+        "Calculation": {
+            "model": "osqp",
+            "symbols": [
+                "ADS.F",
+                "ALV.F",
+                "BAYN.F",
+                "BEI.F"
+            ],
+            "symbol_names": [
+                "adidas AG",
+                "Allianz SE",
+                "Bayer AG",
+                "Beiersdorf AG"
+            ],
+            "risk_weight": 10.0,
+            "esg_weight": 10.0,
+            "start": "2022-08-01",
+            "end": "2022-10-10",
+            "id": 147,
+            "result": []
+        },
+        "Result": {
+            "rate_of_return": {
+                "ADS.F": 0.0,
+                "ALV.F": 0.0,
+                "BAYN.F": 34.27,
+                "BEI.F": 65.73
+            },
+            "esg_rating": {
+                "ADS.F": 13.46,
+                "ALV.F": 16.72,
+                "BAYN.F": 29.88,
+                "BEI.F": 28.97
+            },
+            "volatility": {
+                "ADS.F": 0.0025417845065472365,
+                "ALV.F": 0.0013020043834552056,
+                "BAYN.F": 0.006007783737555586,
+                "BEI.F": 0.0022103124858540045
+            },
+            "objective_value": -1.2697567295179144,
+            "rate_of_return_value": -0.23239186179309934,
+            "risk": 25.873913818877377,
+            "esg_value": 29.281880925170896,
+            "id": 147,
+            "calculation_id": 147
+        }
+    },
+]
+```
+
+The `portfolio_model` component is therefore defining the problem and preparing it for the various algorithms. It gets the stock data from the `Yahoo Finance API` to retrieve on one hand the `Close` prices over a certain period of time and on the other hand the sustainability value of each stock that is requested. To prevent fetching this data again, it is saved to the database file and recalled when needed. For further information on the database read the [DB Documentation](../db/README.md).
+
+The `optimization_backend` handles all the algorithms, identifies to the Azure Quantum and the IBM Qiskit APIs. For the classical algorithms it calls the `pypfopt` and the `osqp` libraries. Of each algorithm a result object is constructed which you can see in the above JSON object. Find further information on the result object and what each field means, in the [Portfolio Use Case Documentation](../portfolio_optimization/README.md#result).
+
+The following Flow Chart visualizes the process from the api point of view to support the understandability of the QuPO backend.
+
```

Comment:
Comments about Flow chart:

- add explanation that all model calculations are based on OSQP model (or change in diagram)
- "if calculation is None" - change to soemthing like "if no existing calulcation"
- add description on arrow from **model** to **Quiskit Job** (model="quiskit")
- move "Configure azure provider" bubble more to the right